pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2839,7 +2839,7 @@ type DisruptionManagementSpec struct {
 	// +optional
 	OSDMaintenanceTimeout time.Duration `json:"osdMaintenanceTimeout,omitempty"`
 
-	// PGHealthCheckTimeout is the time (in minutes) that the operator will wait for the placement groups to become
+	// DEPRECATED: PGHealthCheckTimeout is the time (in minutes) that the operator will wait for the placement groups to become
```

Comment:
Let's remove this setting from all the cluster.yaml and other example manifests

Reply:
done.